components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -38,21 +38,21 @@
           </chef-table-header>
           <chef-table-body>
             <chef-table-row *ngFor= "let server of sortedChefServers$ | async">
-              <chef-table-cell>
+              <chef-table-cell class="align-flex-center">
```

Comment:
Of the 160 instances of `<chef-table-cell>` only 24 of them use `class="align-flex-center"`. Can it really be true that only 15% of these columns need to be centered? That does not feel right. What is special about these 24?

Reply:
Good catch - initially I didn't use when the table doesn't appear to ever be in a scenario where they would have text wrapping applied.  However, probably better to add this to all chef-table cells at the top level and then apply locally when the change is needed.